pkg/rpc/server/server.go

```diff
@@ -1331,7 +1331,8 @@ func (s *Server) sendrawtransaction(reqParams request.Params) (interface{}, *res
 	if err != nil {
 		return nil, response.ErrInvalidParams
 	}
-	return getRelayResult(s.coreServer.RelayTxn(tx), tx.Hash())
+	r, _ := s.coreServer.RelayTxn(tx)
+	return getRelayResult(r, tx.Hash())
```

Comment:
The deal with this error here (in `getRelayResult`), RPC server imports `core` anyway.